scripts/start.py

```diff
@@ -196,25 +196,38 @@ def main(args: Optional[Sequence[str]] = None) -> None:
             )
             initializer.populate_debug_data()
 
-        managed_web_browser = (
-            None if parsed_args.no_browser else
-            servers.create_managed_web_browser(PORT_NUMBER_FOR_GAE_SERVER))
-
-        if managed_web_browser is None:
+        try:
+            managed_web_browser = (
+                None if parsed_args.no_browser else
```

Comment:
I think you should do something like "if parsed_args.no_browser:", do stuff, "else", then all this try/except logic. That way the None case is handled completely externally and is not entangled with all this "launch browser" stuff.

Reply:
Yes, the message can be printed in the "if parsed_args.no_browser:" block. I will make these change tomorrow.

It has been a long day for me and brain is not working at full capacity. I could not understand your suggestions. Apologies for that.

Reply:
No worries, thanks a lot for the quick iterations! I'm turning in for the night too :) See you tomorrow!

Reply:
I have made the changes. Please have a look.